cgi-bin/LJ/Widget/CustomizeTheme.pm

```diff
@@ -223,6 +223,27 @@ sub render_body {
             groupprops => $groups{groupprops}->{customcss},
         );
         $ret .= "</div>";
+        $ret .= "\n\n<script type='text/javascript'>\n";
```

Comment:
For this section, could we do something like this instead:

```perl
$ret .= <<EOF;

<script type='text/javascript'>
    ...
</script>

EOF
```

The reason for doing this is just to make the Perl cleaner / easier to read the underlying code, instead of having to stick in \n's and quotes and stuff. It's also faster to execute, although that's probably not something that matters heh.